docs/How the library works/form-config.md

```diff
@@ -1,580 +0,0 @@
-## Form config
```

Comment:
This went to https://github.com/usds/us-forms-system/pull/112/files#diff-8dbd886edc52aa4c211a3838bff6dfbcR1.